src/Illuminate/Auth/Authenticatable.php

```diff
@@ -4,6 +4,20 @@
 
 trait Authenticatable
 {
+    /**
+     * The column name of the "name" field.
+     *
+     * @var string
+     */
+    protected $authNameKey = 'Name';
+
+    /**
+     * The column name of the "email" field.
+     *
+     * @var string
+     */
+    protected $authEmailName = 'Email';
```

Comment:
Shouldn't it be lower case?

Reply:
Yes. my bad.
I think I'll do this stuff straight in Telescope like it's avatar method.
The idea is to be able to provide Name & Email to show authenticated user when your Column Names aren't as per the defaults.